Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -6683,6 +6683,20 @@ TreatsAnyTextCSSLinkAsStylesheet:
     WebCore:
       default: false
 
+UAVisualTransitionDetectionEnabled:
+  type: bool
+  status: testable
```

Comment:
This seems like "cheating" the WPTs. Can tests tell if there actually is a UA transition?

Reply:
I think the only coverage thus far is that there's this attribute on the object. They're not actually testing the firing of the event. It also seems hard to test that accurately without some WebDriver support as it'll differ across platforms.

Reply:
Right, the WPT test history-traversal/PopStateEvent.html tests very basic idl stuff, unfortunately lowering status to unstable would make us fail it.